examples/testclient/hello/__init__.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+import connexion
+from connexion.lifecycle import ConnexionRequest, ConnexionResponse
+from connexion.problem import problem
+
+logger = logging.getLogger(__name__)
+
+
+async def handle_error(request: ConnexionRequest, ex: Exception) -> ConnexionResponse:
+    """
+    Report an error that happened while processing a request.
+    See: https://connexion.readthedocs.io/en/latest/exceptions.html
+
+    This function is defined as `async` so it can be called by the
+    Connexion asynchronous middleware framework without a wrapper.
+    If a plain function is provided, the framework calls the function
+    from a threadpool and the exception stack trace is not available.
+
+    :param request: Request that failed
+    :parm ex: Exception that was raised
+    :return: ConnexionResponse with RFC7087 problem details
+    """
+    # log the request URL, exception and stack trace
+    logger.exception("Request to %s caused exception", request.url)
```

Comment:
Does this work properly, don't we need to supply the exception as an argument via `exc_info=ex`?

Reply:
The python logger does magic when you call its `exception` method, and that magic breaks when the context changes, like when running the function from a new thread pool. I will test what you suggest - pass the exception - and see if the full stack trace is shown.

Reply:
Thank you for @Ruwann for pointing me to the `exc_info` parameter. Yes, the stack trace appears as expected when that parameter is passed. That seems like a better solution than using `async` to define the exception handler function.